sunpycube/cube/datacube.py

```diff
@@ -639,3 +640,13 @@ def plot_wavelength_slice(self, offset, **kwargs):
             offset, cumul_cube_lengths)
         plot = self[sequence_index].plot_wavelength_slice(cube_index, **kwargs)
         return plot
+
+
+class Slice_helper(object):
+    """Helper class to make slicing in index_as_cube more pythonic"""
```

Comment:
Can you be a little more explicit in you docstring?  Imagine documenting it so one of the IRIS team members, who haven't been following the details of our development, can get a idea of what the purpose of this helper is and how it works.